ra/registration-authority_test.go

```diff
@@ -460,6 +460,22 @@ func TestUpdateAuthorization(t *testing.T) {
 	t.Log("DONE TestUpdateAuthorization")
 }
 
+func TestUpdateAuthorizationExpired(t *testing.T) {
+	_, _, ra, fc, cleanUp := initAuthorities(t)
+	defer cleanUp()
+
+	authz, err := ra.NewAuthorization(AuthzRequest, Registration.ID)
+
+	expiry := fc.Now().Add(-2 * time.Hour)
+	authz.Expires = &expiry
+	test.AssertNotError(t, err, "NewAuthorization failed")
```

Comment:
This `AssertNotError` should come right after the call whose error it is checking.